Library/Homebrew/utils/free_port.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Utils
```

Comment:
Do we anticipate this being used outside `test do`? If not, might make sense to limit the scope a bit?

Reply:
Yep, will limit it to test only.